bridgepoint/prebuild.py

```diff
@@ -305,6 +311,8 @@ def v_irf(self, node, name):
 
     def v_avl(self, node, o_attr, root_v_val):
         s_dt = one(o_attr).S_DT[114]()
+        if s_dt == self.s_dt('same_as<Base_Attribute>'):
```

Comment:
Can this be recursive? see:

https://github.com/xtuml/pyxtuml/blob/fea8471801903f0bb89c11acb16bc59865b2afb5/bridgepoint/gen_xsd_schema.py#L64

Reply:
There is no need for it to be recursive. R113 navigates back to the most base attribute. E.g. if A class has A_ID, B class has B_ID {I, R1}, and C class has C_ID {R2}, If you navigate across R113 from C_ID O_ATTR instance, you get to the O_ATTR instance or A_ID and _not_ the instance for B_ID. BridgePoint always links the farthest back in the chain across R113, so one traversal is enough to get the root.